projects/plugins/jetpack/to-test.md

```diff
@@ -9,13 +9,87 @@
+
+- Go to /wp-admin/admin.php?page=jetpack#/settings
+- A modal should show when you disable the SSO module.
+- Take the survey.
+- Verify the correct userId is added to the survey URL.
+- Verify the survey modal does not display multiple times if you disable/enable the SSO module.
+
+### Newsletter
+
+#### Jetpack sites
+
+- Apply this PR to your JT site
+- Go to the Newsletter Settings page with both flags enabled wp-admin/admin.php?enable-email-settings=true&enable-newsletter-categories=true&page=jetpack#/newsletter
+- Perform tests on all cards and check if the disable state is consistent
+
+- Go to the Newsletter settings page with the feature flag enabled: {your-site}/wp-admin/admin.php?enable-email-settings=true&page=jetpack#/newsletter
+- The Email settings module should be displayed
+- Perform tests on both Featured Image and Excerpt inputs, verify if the data is correctly synced with the settings on wpcom
+- Remove the feature flag to make sure it is not displayed without the flag
+
+#### Contact Forms
+
+The Contact Forms module has been refactored to use the new package instead of the older module shipped with Jetpack. Spend some time testing Contact Forms by adding Contact Form blocks and playing around with their settings. More details in the project thread: pf5801-Aj-p2
+
+### Sharing
+
+This release enables the "Sharing Settings" wp-admin page in offline mode or when Classic (wp-admin) interface is enabled. The "Sharing Settings" page appears different depending on if your site has a blocks or classic theme enabled.
+
+The link in Jetpack > Settings > Sharing > "Configure your sharing buttons" now directs you to the "Sharing Settings" wp-admin page when Classic (wp-admin) interface is enabled.
+
+#### WoA Sites
+
+On a "Default" admin interface (non wp-admin) nothing should have changed.
+On a "Classic" admin interface (wp-admin) the "Sharing settings" wp-admin page should be registered. To test this:
+
+- Go to Appearance > Themes and install / activate a classic theme, e.g.: "Classic"
+- Go to Jetpack > Settings > Sharing and enable sharing buttons.
+- Click the the "Configure your sharing buttons" link that appears at the bottom of the "Sharing buttons" panel.
+- It should direct you to the "Sharing Settings" wp-admin page located here: /wp-admin/options-general.php?page=sharing
+- If your site is using a block theme the "Configure your sharing buttons" will link you to the site editor.
+
+#### Jetpack Sites
+
+Nothing should have changed for WPCOM connected Jetpack sites. Meaning the "Sharing Settings" wp-admin page should not be registered and the "Configure your sharing buttons" should link you to Calypso here: /marketing/sharing-buttons/[site_slug]
+
+### New Sharing Buttons
+
+- Go to Appearance > Themes.
+- Install and activate the Twenty Ten theme.
```

Comment:
Probably copy-pasted, but this seems like a rather old theme to test against. :^)

Reply:
I'm assuming that's a safe bet for a theme that supports classic widgets.